workspace/run2.1/test_agn_config_file_2.1.ddf.json

```diff
@@ -0,0 +1,10 @@
+{"fov": 0.1,
+"unlensed_agn": 1,
+"gzip_threads": 1,
+"descqa_catalog": "cosmoDC2_v1.1.4_image_addon_knots",
+"sed_lookup_dir": "/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4/sedLookup/",
+"db": "/global/projecta/projectdirs/lsst/groups/SSim/DC2/DDF_cadences/ddf_cadence.db",
+"agn_db_name": "/global/cscratch1/sd/jchiang8/desc/Run2.2i/agn_catalog/2019-11-21/cosmoDC2_agn.db",
```

Comment:
We should move this out of my scratch area and into some place like `/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4` and rename the file appropriately.  I created that file using the code that's currently in master via #80 .

Reply:
I've copied that file to 
```
/global/projecta/projectdirs/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4/agn_cosmoDC2_v1.1.4.db
```